backend/cmd/headlamp.go

```diff
@@ -703,19 +743,25 @@ func refreshAndCacheNewToken(oidcAuthConfig *kubeconfig.OidcConfig,
 
 	tk, err := ts.Token()
 	if err != nil {
+		utils.Log(utils.LevelError, nil, err, "Error: failed to get new token")
+
 		return "", err
 	}
 
 	idToken, ok := tk.Extra("id_token").(string)
 	if ok {
 		// update cache
 		if err := cache.Set(context.Background(), fmt.Sprintf("oidc-token-%s", idToken), tk.RefreshToken); err != nil {
+			utils.Log(utils.LevelError, nil, err, "Error: failed to cache refresh token")
```

Comment:
```suggestion
			utils.Log(utils.LevelError, nil, err, "failed to cache refresh token")
```
Make sure to remove the extra "Error" every where the log level is set to Error.